tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/MockTest.java

```diff
@@ -31,20 +31,45 @@
 
     private BMainInstance balClient;
     private String projectPath;
+    private String projectPath2;
 
     @BeforeClass
     public void setup() throws BallerinaTestException {
         balClient = new BMainInstance(balServer);
         projectPath = mockProjectPath.toString();
+        projectPath2 = mockProjectPath2.toString();
     }
 
     @Test
-    public void testAssertTrue() throws BallerinaTestException {
-        String msg = "5 passing";
-        LogLeecher clientLeecher = new LogLeecher(msg);
-        balClient.runMain("test", new String[]{"--all"}, null,
-                new String[]{}, new LogLeecher[]{clientLeecher}, projectPath);
-        clientLeecher.waitForText(20000);
+    public void testFunctionMocking() throws BallerinaTestException {
```

Comment:
Shall we make the function names self explanatory, since there are two for function mocking?

Reply:
Changed the second test function to better reflect what the test does